Dialogue/Support Partner/sp_020000.csv

```diff
@@ -28,11 +28,11 @@ PERSENT_FAV3#2,"""こ、これ、受け取ってくれます？<br>日頃のお
 PERSENT_FAV2#2,"""あの……これ<br>ぼくからのプレゼントです。"""
 PERSENT_FAV1#2,"""PERSENT_FAV1<br>Debag_Error"""
 PERSENT_FAV0#2,"""PERSENT_FAV0<br>Debag_Error"""
-TALK_FAV4#3,"""俺たちの間に言葉はいらない。<br>俺は、お前に尽くすだけだ。"""
-TALK_FAV3#3,"""俺たちの間に言葉はいらない。<br>俺は、お前に尽くすだけだ。"""
-TALK_FAV2#3,"""俺たちの間に言葉はいらない。<br>俺は、お前に尽くすだけだ。"""
-TALK_FAV1#3,"""俺たちの間に言葉はいらない。<br>俺は、お前に尽くすだけだ。"""
-TALK_FAV0#3,"""俺たちの間に言葉はいらない。<br>俺は、お前に尽くすだけだ。"""
+TALK_FAV4#3,"""I do not need words between us. I will do everything for you."""
+TALK_FAV3#3,"""I do not need words between us. I will do everything for you."""
+TALK_FAV2#3,"""I do not need words between us. I will do everything for you."""
+TALK_FAV1#3,"""I do not need words between us. I will do everything for you."""
+TALK_FAV0#3,"""I do not need words between us. I will do everything for you."""
```

Comment:
This line is kind of awkward. Maybe substitute with "There is no need for words between us. I will do anything for you."